packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -219,8 +219,12 @@ abstract class SerializationManager {
               .toList();
         } else if (encodeForProtocol && nonEncodable is ProtocolSerialization) {
           return nonEncodable.toJsonForProtocol();
+        } else if (nonEncodable == null) {
+          return 'null';
```

Comment:
Chore: Remove from this PR.

This to me is a scary change that I don't think we should add in this PR.
If we add it it should be done in isolation, with tests to validate the behavior we are after.

Previously, null would have just returned `null` right? (This proves we don't have any tests for this either...)

Reply:
Good point!